### DISCUSSION TITLE
Figure tabs (idea)

### DISCUSSION THREAD

**Discussion by oscarvdvelde** [JuliaPlots/Makie.jl#3499] [Ideas]
I am developing an application where I would like to show point-based data visualizations in one window, and density based grids in another, but instead of juggling between separate figures, they could appear in the same window under two named tabs.
So, each tab could hold a figure object.

**Comment by math4mad** [comment]
for  now seem's no  tab component  

but  it can  do in quarto note  
 [Quarto - Component Layout](https://quarto.org/docs/interactive/layout.html#tabset-panel)
